host_applications/linux/apps/raspicam/CMakeLists.txt

```diff
@@ -43,9 +43,11 @@ set (COMMON_SOURCES
    RaspiCLI.c
    RaspiPreview.c
    RaspiCommonSettings.c
-   RaspiHelpers.c)
+   RaspiHelpers.c
+   RaspiGPS.c
+   libgps_loader.c)
```

Comment:
You've added it to COMMON_SOURCES, so raspividyuv and raspiyuv will now accept the parameter but do nothing with them.

Reply:
Yes, I thought the benfits of making it common for two of the apps outweighed the fact its not (yet) used in the YUV applications.

Reply:
I'll let you off then.

Reply:
Phew